test/test_categoria.py

```diff
@@ -0,0 +1,16 @@
+from unittest import TestCase
+from src.categoria import Categoria
+
+class TestCategoria(TestCase):
+
+    def test_quando_CriaCategoria_retorna_SemErro(self):
+        categoria = Categoria("Machine Learning")
+        self.assertIsNotNone(str(categoria))
```

Comment:
Aqui você tá testando basicamente o __str__ não? o str vai chamar o __str__ e aparentemente o teste sempre vai passar. Se for esse caso o __set__ da categoria não tá sendo utilizado para nada além do teste, sendo assim eu removeria os dois, o teste e o __str__

Reply:
Quando fiz esse teste estava pensando no cenário "feliz" de se criar uma nova Categoria dentro da coleção de de categorias.

Acredito que eu deva mudar o nome do teste para "test_quando_CriaCategoria_retorna_NomeDaCategoria" e testar se ela salvou o nome que eu passei quando criei a Categoria.